Services/AdvancedEditing/classes/class.ilObjAdvancedEditing.php

```diff
@@ -134,9 +132,7 @@ public static function _getUsedHTMLTagsAsString(string $a_module = "") : string
      */
     public static function _getRichTextEditor() : string
     {
-        $setting = new ilSetting("advanced_editing");
-        $js = $setting->get("advanced_editing_javascript_editor", "0");
-        return $js;
+        return (new ilSetting("advanced_editing"))->get("advanced_editing_javascript_editor", "0");
```

Comment:
ok, i don't like this spaghetti in one line style.

Reply:
@alex40724 Should I revert this?

Reply:
No, thanks. Maybe avoid it in the future.